docs/content/concepts/executor.en.md

```diff
@@ -4,19 +4,65 @@ draft: false
 weight: 23
 ---
 
-When you create a function, you can specify an executor for a function. An executor controls how function pods are created and what capabilities are available for that executor type.
+A key aspect of performance latency in any high-throughput Serverless application is activation latency for incoming functions, since every single function effectively may require instantiation of a new container in a naive implementation.
```

Comment:
The first half of the sentence is saying that a key aspect of latency is latency? I don't get it.

Are you trying to describe the cold start problem?  This is not what I would start this doc with.  Also, the text you removed is quite important -- the executor is what decides how functions are executed, and we need to tell users what it is.